lib/kafo/configuration.rb

```diff
@@ -47,6 +52,7 @@ def initialize(file, persist = true)
       end
 
       @config_dir = File.dirname(@config_file)
+      @scenario_id = Configuration::get_scenario_id(@config_file)
```

Comment:
I guess you meant `Configuration.get_scenario_id`, it's regular class method

Reply:
Isn't this the way to call it statically?

Reply:
Updated